src/modules/middleware/validate-middleware.js

```diff
@@ -0,0 +1,18 @@
+const {check, validationResult} = require('express-validator');
+
+module.exports = taskValidator = (req, res, next) => {
+    check('text')
+    .exists()
+    .not()
+    .isEmpty(),
+    check('isCheck')
+    .exists()
+    .not()
+    .isEmpty()
+    const error = validationResult(req);
+    if (error.isEmpty()) {
+        next();
+    } else {
+        res.status(400).send('Bad request. Validation error');
+    }
```

Comment:
```
    const error = validationResult(req);
    if (error.isEmpty()) {
        next();
    } else {
        res.status(400).send('Bad request. Validation error');
    }
    
```
this can be moved to a separate middleware